agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/localstorage/LocalFileLoaderTests.java

```diff
@@ -165,17 +175,16 @@ public void testWriteGzipRawByte() throws IOException {
     // write gzipped bytes[] to disk
     byte[] result = byteArrayOutputStream.toByteArray();
     LocalFileCache cache = new LocalFileCache();
-    LocalFileWriter writer = new LocalFileWriter(cache);
+    LocalFileWriter writer = new LocalFileWriter(cache, false);
     writer.writeToDisk(singletonList(ByteBuffer.wrap(result)));
 
     // read gzipped byte[] from disk
-    LocalFileLoader loader = new LocalFileLoader(cache);
+    LocalFileLoader loader = new LocalFileLoader(cache, false);
     byte[] bytes = readTelemetriesFromDiskToBytes(loader);
 
     // ungzip
     ByteArrayInputStream inputStream = new ByteArrayInputStream(result);
-    // TODO (heya) does ungzip need to be larger?
```

Comment:
gzip byte[] is always bigger than the origin byte[].  it works previously was because the text was very small.  gzip happens to be equal to the ungzip.. when the text becomes longer and longer, the gzip byte size can be triple of the original one.